### PR TITLE
[WPE-276][wificontrol] Ensure config store path exists.

### PR DESCRIPTION
**Pull request by pawel-stanek-ml**

### CODE REVIEW

File: WifiControl/WifiControl.cpp [hunk 1]
```diff
@@ -75,6 +75,11 @@ namespace Plugin
         _skipURL = static_cast<uint8_t>(service->WebPrefix().length());
         _service = service;
 
+        if (Core::Directory(service->PersistentPath().c_str()).CreatePath())
+            _configurationStore = service->PersistentPath() + "wpa_supplicant.conf";
+        else
+            TRACE_L1("Config directory %s doesn't exist and could not be created!\n", service->PersistentPath().c_str());
```
Comment (pwielders) [inline WifiControl/WifiControl.cpp]: This is serious and should be reported using a SYSLOG.

Comment (pawel-stanek-ml) [reply WifiControl/WifiControl.cpp]: SYSLOG it is :)

File: WifiControl/WifiControl.cpp [hunk 2]
```diff
@@ -75,6 +75,11 @@ namespace Plugin
         _skipURL = static_cast<uint8_t>(service->WebPrefix().length());
         _service = service;
 
+        if (Core::Directory(service->PersistentPath().c_str()).CreatePath())
+            _configurationStore = service->PersistentPath() + "wpa_supplicant.conf";
+        else
+            SYSLOG(Logging::Startup, ("Config directory %s doesn't exist and could not be created!\n", service->PersistentPath().c_str()));
```
Comment (pawel-stanek-ml) [inline WifiControl/WifiControl.cpp]: Syslog here I guess?

Comment (pawel-stanek-ml) [reply WifiControl/WifiControl.cpp]: Ah. No. There's no such cathegory. Dunno if Startup is proper one

Comment (pwielders) [reply WifiControl/WifiControl.cpp]: Yip Startup is a proper one

**Pull request merged**